### ISSUE TITLE
Using wrong axis in TF implementation

### ISSUE THREAD

**Issue by christiantinauer** [sebastian-lapuschkin/lrp_toolbox#3]
Hi,

I think there is a bug in the TF implementation for convolution. Expanding is done at differrent positions. In my opinion the second one is correct (expand_dims(axis=3)). From the repository:

**def _simple_lrp(self,R):**
`result = tf.reduce_sum((Z/Zs) * tf.expand_dims(self.R[:,i:i+1,j:j+1,:], 1), 4)`

compared to 

**def _flat_lrp(self,R):**
`result = tf.reduce_sum((Z/Zs) * tf.expand_dims(self.R[:,i:i+1,j:j+1,:], 3), 4)`

The pure Python implementation always expands at the dimensions at position 3. Is this a bug or am I overlooking something?

**Comment by VigneshSrinivasan10** [comment]
@christiantinauer : Thanks for your interest. The branch of TF implementation is a work in progress and hence the several checks and updates are underway. We will soon release the first version and we will keep you posted on the developments.
